packages/account/src/Sections/Security/ConnectedApps/connected-apps.tsx

```diff
@@ -71,24 +69,9 @@ const ConnectedApps = observer(() => {
                     <div className='connected-apps__content--wrapper'>
                         <ConnectedAppsInfo />
                         {is_mobile ? (
-                            <div className='connected-apps__list--wrapper'>
-                                {connected_apps.map(connected_app => (
-                                    <DataListTemplate
-                                        key={connected_app.app_id}
-                                        data_source={connected_app}
-                                        handleToggleModal={handleToggleModal}
-                                    />
-                                ))}
-                            </div>
+                            <DataListTemplate connected_apps={connected_apps} handleToggleModal={handleToggleModal} />
```

Comment:
what if connected_apps doesnt have any item? Do we still need to render this components?

Reply:
in line 68, we are checking if connected_apps has required length before rendering the these.
if connected_apps is empty, another component will be rendered.